lib/tasks/tsheets/import_users.rake

```diff
@@ -0,0 +1,13 @@
+namespace :tsheets do
+  task import_users: :environment do
+    service = TsheetsService.new
+    result = service.sync_users
+    result[:synced].each do |user|
+      puts "#{user.first_name} #{user.last_name} Synced"
```

Comment:
I often use puts in rake tasks to output activity to the console - you can see what it's doing without having to tail the relevant log in that case, I don't think that's hacky?

It's just feedback on what's happening, not logging for debug purposes.